Tests/ImagineEngineTests/ActorTests.swift

```diff
@@ -325,6 +325,20 @@ final class ActorTests: XCTestCase {
         XCTAssertEqual(velocities, [Vector(dx: 100, dy: 0), Vector(dx: 100, dy: 50)])
     }
 
+    func testObservingRotationChange() {
+        var noValueTriggerCount = 0
```

Comment:
Nitpick: Since there's no `valueTriggerCount` I think this can simply be named `triggerCount` 🙂

Reply:
Ahh, yes, good point! The evil of c&p... 😄
Super fun to be working on ImagineEngine!